evidence/cli/command_controller_test.go

```diff
@@ -0,0 +1,101 @@
+package cli
+
+import (
+	"github.com/golang/mock/gomock"
+	"github.com/jfrog/jfrog-cli-core/v2/common/commands"
+	"reflect"
+	"testing"
+	"unsafe"
+
+	"github.com/jfrog/jfrog-cli-core/v2/plugins/components"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateEvidence_Context(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockExec := func(command commands.Command) error {
```

Comment:
The override can be applied once for all the test cases:
```go
	execFunc = func(command commands.Command) error {
		return nil
	}
```
(no need to repeat the same `mockExec` and defer the restoration.

Reply:
done